# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../musl/pkg/util-vserver/util-vserver.patch
# Copyright (C) 2014 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

--- util-vserver-0.30.216-pre3054/compat.h.orig	2013-10-31 14:44:25.000000000 +0200
+++ util-vserver-0.30.216-pre3054/compat.h	2014-03-14 18:59:20.750464540 +0200
@@ -93,7 +93,7 @@
 #endif
 
 
-#if defined(__dietlibc__)
+#if !defined(__GLIBC__ )
   #define TEMP_FAILURE_RETRY(expression)				\
   (__extension__							\
    ({ long int __result;		
